tests/libvmi/vmi.go

```diff
@@ -171,6 +171,23 @@ func WithSEV() Option {
 	}
 }
 
+// WithRealtimeConfiguration configures the CPU and Hugepages for realtime tests
+func WithRealtimeConfiguration(memory, realtimeMask string) Option {
```

Comment:
Thanks for this PR. It makes the test easier to understand.

I don't think that Realtime is "important enough" to be placed in `vmi.go`. I would like to keep this file short and simple to read, including only content that is known to be widely usable by tests. I'm not sure that the single use of `WithRealtimeConfiguration` in a single test merits to clutter `libvmi` with; I think it can perfectly fit in the realtime test file. However, if others (e.g @iholder-redhat @EdDev ) think that this should go under `tests/libvmi`, please consider creating `tests/libvmi/realtime`.

Reply:
`WithRealtimeConfiguration` is at least shared by `tests/performance/realtime.go` and `tests/realtime/realtime.go`. `tests/libvmi/realtime` sounds like a good idea. Let's wait for more feedback.

Reply:
> `WithRealtimeConfiguration` is at least shared by `tests/performance/realtime.go` and `tests/realtime/realtime.go`. 

Oh, I've missed that. So I would support placing it in `tests/libvmi/realtime`.

Reply:
This is an opinionated collection of parameters.
Adding each parameter independently is generic enough to fit libvmi, but as a collection I think it is not.

I would prefer repeating the options, it makes things visible. But in specific packages it may make sense to define it as a collection.

Reply:
Agreed, I pushed new changes. Can you have a look please?

Reply:
I think most of the options will not be realtime specific, more like performance/cpu specific.

Reply:
I agree with @xpivarc.
A VM doesn't have to be "real-time" in order to define a CPU model / guest memory amount.
Both `cpu.go` or `performance.go` are very good suggestions imo

Reply:
Good suggestion! Applied.